### ISSUE TITLE
Apple Mail Crashes

### ISSUE THREAD

**Issue by alanremen** [CocoaLumberjack/CocoaLumberjack#756]
I have a brand new MacBook Pro - I use Apple Mail with the Daylite Mail Assistant plugin. I continuously experience Apple Mail crashing and I get this warning flag:

**Mail quit unexpectedly while using the CocoaLumberJack plug-in.**

I have repeatedly uninstalled and reinstalled Daylite - didn't solve the problem. Apple Tech Support can't find the plugin on my system and suggest I contact you.

This is extremely disruptive - we need some help here.

### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / osx / tvos / watchos
 Platform Version        | e.g. 8.0
 CocoaLumberjack Version | e.g. 2.3.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
I don't understand how this is related to our library
